ci-operator/step-registry/ipi/install/install/stableinitial/ipi-install-install-stableinitial-ref.yaml

```diff
@@ -26,6 +26,9 @@ ref:
   - name: OPENSHIFT_INSTALL_GATHER_BOOTSTRAP
     default: "true"
     documentation: "Collect bootstrap log bundle, even on success."
+  - name: FIPS_ENABLED
+    default: "false"
+    documentation: "Indicates a FIPS-enabled cluster, to disable FIPS host checks in the installer."
```

Comment:
this passes the env var through on _all_ 2000+ jobs that use this step, but given that it supplies the same default as the script already uses, i think it's unlikely it will have any effect. i rehearsed one of the jobs just for a sanity check.

the alternative would be to create a parallel step that's exactly the same but recognizes FIPS. that seems like clutter to me.